Examples/ConstructingPackets/Main.cs

```diff
@@ -36,6 +36,7 @@ public static void Main(string[] args)
 
             // and print out the packet to see that it looks just like we wanted it to
             Console.WriteLine(ethernetPacket.ToString());
+            Console.ReadKey(true);
```

Comment:
Ahh. I normally run the applications from the console (on linux) but I see what you mean. Windows I think will just close the application window out once the application exits, and it will do so immediately without that line. Its probably best to stay consistent until we can discuss and agree upon the other change. Is it so unusual to run applications from the console on windows?